### ISSUE TITLE
Issues with voicingIcon.png

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/scenery-phet#695]
images/voicing-icon.png (EDIT: now voicingIcon.png, see below) was added to scenery-phet on 9/4/2020.   It was added by @jessegreenberg, and the license.json file indicates that the artwork was created by @terracoda.

Two issues:

(1) There is no corresponding file in scenery-phet/assets/.  For all image files, the original file (typically an Adobe Illustrator .ai file) that was used to create the PNG file is required to be commited to GitHub.  This ensures that we have access to the .ai file if/when changes to artwork need to be made.

(2) There are no uses of this image in the master branches of any PhET sim repo or dependency.  Is it used in a branch?

**Comment by pixelzoom** [comment]
Note that voicing-icon.png has been renamed to voicingIcon.png in https://github.com/phetsims/scenery-phet/issues/694, so that all images in this repository follow a consistent naming convention, compatible with https://github.com/phetsims/chipper/issues/1060#issuecomment-898662534.

**Comment by terracoda** [comment]
@amanda-phet and I worked on the concept together. @amanda-phet should have the file.

**Comment by terracoda** [comment]
@pixelzoom, we are talking about the icon used on the website, right?

**Comment by pixelzoom** [comment]
@terracoda Sorry, I have no idea where this icon is used.  That is one of the problems mentioned above:

> (2) There are no uses of this image in the master branches of any PhET repo. Is it used in a branch?

If it's used on the website, then it does not belong in this repository.

It looks like this:

<img width="298" alt="screenshot_1183" src="https://user-images.githubusercontent.com/3046552/129620846-d6179d6b-7d2e-486b-98e5-fa109e30418e.png">

**Comment by terracoda** [comment]
Thanks for the clarification @pixelzoom. That version of the voicing icon was used in earlier versions before we had the Voicing Toolbar. We no longer use that icon directly in the Voicing feature which is likely why you only see it in a branch.

@jessegreenberg, can you confirm that this was the icon we used for the Voicing accordion before we had designed the Toolbar?

I think we have intentionally kept older versions of the Voicing feature because it went through several paradigms of thinking before we arrived at the final design. We want to keep versions of these paradigms because we need to write a paper about the design them.

What's the procedure to document this image. It's not exactly the final icon used on the website. That said @amanda-phet may still might have the design file. @amanda-phet, do you happen to still have the appropriate image file of this version of the icon?

**Comment by pixelzoom** [comment]
> We no longer use that icon directly in the Voicing feature which is likely why you only see it in a branch.

I'm not seeing it in a branch. I'm seeing it in master.  I asked if it was used in a branch, because we don't typically have unused images checked in to a repository. If it's not used in master or any branch that depends on master, I recommend deleting voicingIcon.png from master.   It can always be recovered later if it's needed - that the nature of GitHub.

@jessegreenberg could you please take this over, and figure out what needs to be done here?

**Comment by terracoda** [comment]
Thanks again @pixelzoom for the clarification. @jessegreenberg and will sort this out.

**Comment by terracoda** [comment]
@jessegreenberg and I discussed the voicingIcon.png image. It is not currently being using in the Voicing feature, so it should not be checked in.

Assigning to @jessegreenberg to remove it.

**Comment by jessegreenberg** [comment]
Thanks for reporting, this image should not be checked in anymore, Ill delete it. I read through the comments in this thread to make sure I didn't miss anything but there is no reason it should be checked into master. It also isn't used in any feature branches.

**Comment by pixelzoom** [comment]
Raising priority to high.  This issue has been languishing and should be wrapped up.

**Comment by jessegreenberg** [comment]
voicingIcon.png has been removed from scenery-phet, closing.
